scripts/create-azure-openai-resources.sh

```diff
@@ -156,5 +167,6 @@ echo "-----------------------------------------"
 echo "Here are some key values you may need to use in your application's configuration:"
 echo
 echo "API key (quarkus.langchain4j.azure-openai.api-key): $AZURE_OPENAI_KEY"
+echo "API endpoint: $AZURE_OPENAI_ENDPOINT"
```

Comment:
We don't need/care about the endpoint anymore, unless you think simply outputting it is valuable? It isn't used anywhere.

Reply:
Ok. Got rid of it in https://github.com/quarkusio/quarkus-super-heroes/pull/570/commits/248f7999774b3936dce33787d05a4e2f74dad096